config/locales/de.yml

```diff
@@ -542,26 +542,6 @@ de:
   time:
     formats:
       long_ordinal: "%e %B %Y %l:%M%P"
-  topical_event:
-    headings:
-      detailed_guidance:
-      documents:
-      featured:
-      organisations:
-      travel_advice:
-    latest_activity: Jüngste Aktivität bei %{event}
```

Comment:
We need to check that we are not missing this 'latest_activity' label anywhere in collections - was being used for atom feed here, but we haven't added this or any translations to collections atm.